mlos_bench/mlos_bench/storage/base_storage.py

```diff
@@ -294,8 +294,8 @@ def config(self, global_config: Optional[Dict[str, Any]] = None) -> Dict[str, An
 
         @abstractmethod
         def update(self, status: Status, timestamp: datetime,
-                   metrics: Optional[Union[Dict[str, float], float]] = None
-                   ) -> Optional[Dict[str, float]]:
+                   metrics: Optional[Union[Dict[str, Any], float]] = None
+                   ) -> Optional[Dict[str, Any]]:
```

Comment:
nit:
```suggestion
                   ) -> Optional[Dict[str, TunableValue]]:
```
?

Reply:
It's a DBMS, so should be only strings, ints, and floats, right?  Maybe also nulls ...

Reply:
good catch! fixing it now

Reply:
mmm, maybe not. ☹️ Turned out, using `TunableValue` type spreads out in all directions to the `Environment` and `Optimizer` classes. Let's keep it small for this PR and do the bigger type change later, because I am also not 100% sure that `TunableValue`, which is a type for *input* configuration parameters should be also used for the *output* metrics and telemetry. Maybe we should have a different type there, e.g., `MetricValue`, even it is identical to the `TunableValue`